results/main.py

```diff
@@ -0,0 +1,347 @@
+from toolz.curried import filter as filter_
```

Comment:
Looks like a lot of code duplicated from `pypfhub/main.py`. Can this be consolidated?

Reply:
This was legacy. Deleted now.